config/daemon/systemd.md

```diff
@@ -96,18 +96,34 @@ you need to add this configuration in the Docker systemd service file.
     ```
 
 3.  If you have internal Docker registries that you need to contact without
-    proxying you can specify them via the `NO_PROXY` environment variable:
-
+    proxying you can specify them via the `NO_PROXY` environment variable.
+
+    The `NO_PROXY` variable specifies a string that contains comma-separated values for hosts that should be excluded from proxying. 
+    These are the options you have to specify the excluded hosts: 
+    * IP address prefix (`1.2.3.4`) or in CIDR notation (`1.2.3.4/8`)    
+    * Domain name, or a special DNS label (`*`)
+      * A domain name matches that name and all subdomains. A domain name with
```

Comment:
I think we don't need this bullet to be one level deeper; and we may be able to get rid of the code _block_ below. I gave this a quick go, and posted a suggestion in a comment on a few lines up 🤗